packages/bsky/tests/views/__snapshots__/follows.test.ts.snap

```diff
@@ -5,9 +5,9 @@ Object {
   "cursor": "0000000000000::bafycid",
   "followers": Array [
     Object {
-      "avatar": "https://bsky.public.url/img/avatar/plain/user(3)/cids(0)@jpeg",
+      "avatar": "https://bsky.public.url/img/avatar/plain/user(1)/cids(0)@jpeg",
```

Comment:
any idea why these snaps updated?  they look totally the same to me, just the numbers changed 🧐

Reply:
Oh yeah, I meant to make a note of that.  I was able to verify it's because the response fields now come in a different order.  Jest canonicalizes the output for the snapshots, but the util that replaces cids with `cid(x)`, etc. depends on the order of `Object.entries()`.  Surprised we haven't run into this more often!

Reply:
ah nice 👍